### ISSUE TITLE
RadioSelect should have a .focus-method

### ISSUE THREAD

**Issue by bjoerge** [sanity-io/sanity#527]
RadioSelect is currently not focusable.

Not exactly sure whether `.focus()` should trigger focus on the first radio input in it, or if we should add tabIndex=0 and set focus on a wrapper element.

**Comment by kristofferj** [comment]
I think it should focus the first item by default.

**Comment by kristofferj** [comment]
https://github.com/sanity-io/sanity/pull/711

**Comment by kristofferj** [comment]
This is now in a PR, and it focuses the first item  if no selcted, or the currently selected one.
